airflow/www/static/js/dag/details/taskInstance/Logs/LogBlock.tsx

```diff
@@ -59,10 +67,43 @@ const LogBlock = ({ parsedLogs, wrap, tryNumber }: Props) => {
     }
   };
 
+  const onClick = (e: React.MouseEvent<HTMLElement>) => {
+    const target = e.target as HTMLElement;
+    if (target.id?.endsWith("_unfold")) {
+      const gId = target.id.substring(0, target.id.length - 7);
```

Comment:
I'm not the biggest fan of just hard-coding the number `7`. Perhaps we could make `const unfoldString = `_unfold` and we can use `unfoldString.length`. Same for fold

Reply:
Good point. Actually very good point. Adjusted in all places with constants.